Modules/System/Convert/app.json

```diff
@@ -0,0 +1,33 @@
+{
+    "id":  "ad94bc69-99fa-4bf0-85b3-6dbc72bd1543",
+    "name":  "Convert",
```

Comment:
That is a very generic name you chose :-) If someone wants to build a picture converter, should that get added to this module? Or currency conversion? I think this module would get too wide.
I think we should try to find a module name, which is specific, yet broad enough to fit more functions than the convert function of the encoding class. Finding the right balance... That is not easy, when all we've got is a single function you'd like to add :-D
So, Encoding is an option... Then the module could hold all functionality which concerns encoding. But again, maybe a little broad? "String Encoding" - now it's narrowed down. Maybe too narrow? "String Manipulation" is another option; it could turn into a module, which has all sorts of functions to manipulate strings.
I think I need to give this a little thought. If you have any good ideas, just shoot :-)

Reply:
String Converter? Or should it have DotNet in front of it, to make it more specific? 
DotNet String Manipulation? 
DotNet String Converter?

Reply:
Definitely not anything with DotNet in front of it. Whether you internally in the module use .NET, AL or something else entirely, should not matter to the module itself at all. The module simply does, what it's API specifies; and the API won't give away, that .NET is used.
I'll chew on it a day :-)

Reply:
Hi Jesper,

Did you give it some thought? Is the rest of the Pull request approved? :)

Reply:
The code as such is fine :-) I will work on the module name - no action item for you. We do have 2 more weeks, before we cut the next round of minor releases. I'll work it out in time to catch that release train!

Reply:
What about calling the module _Encoding_?

Reply:
Ok, let's go with Encoding as module name.